common/on_action/wc_aging_penalties.txt

```diff
@@ -0,0 +1,165 @@
+﻿on_birthday = {
+	on_actions = {
+		on_birthday_aging
+	}
+}
+on_birthday_aging = {
+	trigger = {
+		30 <= { value = age divide = racial_age_multiplier_value }
+	}
+
+	effect = {
+		if = {
+			limit = {
+				has_trait_with_flag = can_not_get_sick
+				NOT = { has_trait = being_light }
+				NOT = { has_trait = being_void }
+			}
+			clear_age_penalties = yes
+		}
+		else = {
+			save_temporary_scope_value_as = {
+				name = my_human_age_multiplier
+				value = racial_age_multiplier_value
+			}
+			
+			if = {
```

Comment:
If you implement `stacking = yes` on the modifier, you can replace this else/if logic with a math formula. The input of the formula should be your age, the output should be "stacks" of the modifier.